File: backend/booking-service/src/main/java/it/roadies/booking_service/data/dto/response/BookingHomeResponse.java
package it.roadies.booking_service.data.dto.response;

import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
public class BookingHomeResponse {
    private UUID bookingId;
    private UUID principalId;
    private String travelName;
    private Integer peopleCount;
    private BigDecimal totalPrice;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
    private DepartureType departureType;
}

File: android-app/app/src/main/java/it/roadies/android_app/client/models/booking/BookingHomeResponse.kt
package it.roadies.android_app.client.models.booking

import com.google.gson.annotations.SerializedName
import java.math.BigDecimal
import java.time.LocalDateTime
import java.util.UUID

data class BookingHomeResponse(
    @SerializedName("bookingId")
    val bookingId: UUID? = null,
    @SerializedName("principalId")
    val principalId: UUID? = null,
    @SerializedName("travelName")
    val travelName: String? = null,
    @SerializedName("peopleCount")
    val peopleCount: Int? = null,
    @SerializedName("totalPrice")
    val totalPrice: BigDecimal? = null,
    @SerializedName("startDate")
    val startDate: LocalDateTime? = null,
    @SerializedName("endDate")
    val endDate: LocalDateTime? = null,
    @SerializedName("departureType")
    val departureType: DepartureType? = null
    )
